worlds/bomb_rush_cyberfunk/Items.py

```diff
@@ -0,0 +1,552 @@
+    {'name': "Outfit (Tryce - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Bel - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Bel - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Vinyl - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Vinyl - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Solace - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Solace - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Felix - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Felix - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Rave - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Rave - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Mesh - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Mesh - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Shine - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Shine - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Rise - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Rise - Winter)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Coil - Autumn)",
+        'type': BRCType.Outfit},
+    {'name': "Outfit (Coil - Winter)",
+        'type': BRCType.Outfit},
+
+    # Characters
+    {'name': "Tryce",
+        'type': BRCType.Character},
+    {'name': "Bel",
+        'type': BRCType.Character},
+    {'name': "Vinyl",
+        'type': BRCType.Character},
+    {'name': "Solace",
+        'type': BRCType.Character},
+    {'name': "Rave",
+        'type': BRCType.Character},
+    {'name': "Mesh",
+        'type': BRCType.Character},
+    {'name': "Shine",
+        'type': BRCType.Character},
+    {'name': "Rise",
+        'type': BRCType.Character},
+    {'name': "Coil",
+        'type': BRCType.Character},
+    {'name': "Frank",
+        'type': BRCType.Character},
+    {'name': "Rietveld",
+        'type': BRCType.Character},
+    {'name': "DJ Cyber",
+        'type': BRCType.Character},
+    {'name': "Eclipse",
+        'type': BRCType.Character},
+    {'name': "DOT.EXE",
+        'type': BRCType.Character},
+    {'name': "Devil Theory",
+        'type': BRCType.Character},
+    {'name': "Flesh Prince",
+        'type': BRCType.Character},
+    {'name': "Futurism",
+        'type': BRCType.Character},
+    {'name': "Oldhead",
+        'type': BRCType.Character},
+
+    # REP
+    {'name': "8 REP",
+        'type': BRCType.REP},
+    {'name': "16 REP",
+        'type': BRCType.REP},
+    {'name': "24 REP",
+        'type': BRCType.REP},
+    {'name': "32 REP",
+        'type': BRCType.REP},
+    {'name': "48 REP",
+        'type': BRCType.REP},
+
+    # App
+    {'name': "Camera App",
+        'type': BRCType.Camera}
+]
+
+
+group_table: Dict[str, Set[str]] = {
+    "graffitim": ["Graffiti (M - OVERWHELMME)",
```

Comment:
The typing says Dict[str, Set[str]], but it appears this is a Dict[str, List[str]]

Reply:
Oops, you're right. I always get the syntax for them confused with each other